File: quan.cs/Test/Config/Item/UseEffect3.cs
using Newtonsoft.Json.Linq;
using Quan.Utils;

namespace Test.Config.Item
{
    /// <summary>
	/// 使用效果3<br/>
	/// 代码自动生成，请勿手动修改
	/// </summary>
    public class UseEffect3 : UseEffect
    {
        public readonly int ccc;


        public UseEffect3(JObject json) : base(json)
        {
            ccc = json["ccc"]?.Value<int>() ?? default;
        }

        public new static UseEffect3 Create(JObject json)
        {
            return new UseEffect3(json);
        }

        public override string ToString()
        {
            return "UseEffect3{" +
                   "aaa=" + aaa.ToString2() +
                   ",ccc=" + ccc.ToString2() +
                   '}';
        }
    }
}

File: quan.java/quan-config/src/test/java/quan/config/item/UseEffect3.java
package quan.config.item;

import com.alibaba.fastjson.*;

/**
 * 使用效果3<br/>
 * 代码自动生成，请勿手动修改
 */
public class UseEffect3 extends UseEffect {

    public final int ccc;


    public UseEffect3(JSONObject json) {
        super(json);

        this.ccc = json.getIntValue("ccc");
    }

    public static UseEffect3 create(JSONObject json) {
        return new UseEffect3(json);
    }

    @Override
    public String toString() {
        return "UseEffect3{" +
                "aaa=" + aaa +
                ",ccc=" + ccc +
                '}';

    }

}
